ScalaScript.sublime-syntax

```diff
@@ -67,8 +78,24 @@ contexts:
       scope: constant.numeric.integer.scalascript
 
     # Booleans
-    - match: '(?i)\b(true|false|on|off)\b'
+    - match: '(?i)\b(True|False|On|Off)\b'
```

Comment:
I know the Designer converts boolean values true/false into On/Off on the next save...

Was there once the possibility to specifiy true/false explicitly in an older version. I only know about Scala Designer 6 and later.